slither.config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "detectors_to_exclude": "assembly,low-level-calls,solc-version,missing-zero-check,timestamp",
+    "filter_paths": "(contracts/(auth/AxelarAuthWeighted|test|util/Proxy|.*ERC20.*|.*Deposit(Handler|Receiver).*|EternalStorage.*)|node_modules/)"
```

Comment:
`auth/AxelarAuthWeighted` contract can be modified (so feel free to resolve those) for this release

Reply:
Commented in place instead 👍